src/gen_dragonfruit/gd_dev_apogee_sp215.cpp

```diff
@@ -4,13 +4,13 @@ void gd_dev_apogee_sp215_open(void){
     Wire.begin(9600);
```

Comment:
This function is wrong - what should `Wire.begin` accept as an argument and why?